### PR TITLE
[release] 20220406

### PR DESCRIPTION
**Pull request by jiqiang90**

### CODE REVIEW

File: packages/cli/CHANGELOG.md [hunk 1]
```diff
@@ -8,7 +8,9 @@ All logs must start with the format: [x.y.z] - yyyy-mm-dd
 
 ## [Unreleased]
 
-
+## [0.24.0] - 2022-04-06
+### Added
+- Add support for manifest 1.0.0 (#845)
```
Comment (ianhe8x) [inline packages/cli/CHANGELOG.md:13]: migrate will now upgrade project to 1.0.0

File: packages/node/CHANGELOG.md [hunk 2]
```diff
@@ -6,7 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-
+## [0.33.0] - 2022-04-06
+### Added
+- Add support for handle manifest 1.0.0 (#845)
```
Comment (ianhe8x) [inline packages/node/CHANGELOG.md:11]: Can you add a link to manifest 1.0.0 docs?

Comment (jiqiang90) [reply packages/node/CHANGELOG.md:11]: We don't have document for manifest 1.0.0 yet, we can update it once available

File: packages/query/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@subql/query",
-  "version": "0.14.1-1",
+  "version": "0.15.0",
```
Comment (ianhe8x) [inline packages/query/package.json:3]: since no new features, we can use patch release

**Pull request merged**